File: documentation-examples-kotlin/src/kotlin/dev/typr/foundations/docs/postgresql/UuidType.kt
package dev.typr.foundations.docs.postgresql

import dev.typr.foundationskt.*
import dev.typr.foundationskt.data.*
import java.util.UUID

@Suppress("unused")
class UuidType {
    //start
    val uuidType: PgType<UUID> = PgTypes.uuid
    //stop
}

File: documentation-examples-scala/src/scala/dev/typr/foundationssc/docs/postgresql/UuidType.scala
package dev.typr.foundationssc.docs.postgresql
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*

import java.util.UUID

@SuppressWarnings(Array("unused"))
object UuidType:
  // start
  val uuidType: PgType[UUID] = PgTypes.uuid
  // stop
